pkg/api/secretbootstrap/secretboostrap.go

```diff
@@ -18,7 +18,8 @@ const (
 	AttributeTypePassword AttributeType = "password"
 )
 
-type BitWardenContext struct {
+type ItemContext struct {
+	Item                 string                 `json:"item"`
```

Comment:
Why did we add a new field? Please just rename the existing field without changing the json annotation

Reply:
I wanted to create a new json but still support the old one to keep backwards compatibility, and change the config without breaking anything.

Reply:
Ah sorry, I missed that, the fact that you changed all existing fixtures and tests to use the new format confused me. Can we not do that to:
* Reduce the size of this PR
* Proof that the old format our config is in still works?

Reply:
Well, eventually we will end up changing them, so in this PR I only change their names and load them from the old values during the unmarshaling. The old version should be working fine.  I can try to see what it takes to make this PR smaller and better reviweablt.